group_vars/all.yml

```diff
@@ -39,38 +39,38 @@ sync:
+  acs_zip_url: "{{ nexus_repository.enterprise_releases }}/alfresco-content-services-distribution/{{ acs.version }}/alfresco-content-services-distribution-{{ acs.version }}.zip"
+  acs_zip_sha1_checksum: "{{ lookup('url', '{{ nexus_repository.enterprise_releases }}/alfresco-content-services-distribution/{{ acs.version }}/alfresco-content-services-distribution-{{ acs.version }}.zip.sha1', username=lookup('env', 'NEXUS_USERNAME'), password=lookup('env', 'NEXUS_PASSWORD')) }}"
+  adw_zip_url: "{{ nexus_repository.enterprise_releases }}/alfresco-digital-workspace/{{ adw.version }}/alfresco-digital-workspace-{{ adw.version }}.zip"
+  adw_zip_sha1_checksum: "{{ lookup('url', '{{ nexus_repository.enterprise_releases }}/alfresco-digital-workspace/{{ adw.version }}/alfresco-digital-workspace-{{ adw.version }}.zip.sha1', username=lookup('env', 'NEXUS_USERNAME'), password=lookup('env', 'NEXUS_PASSWORD')) }}"
+  search_zip_url: "{{ nexus_repository.releases }}/alfresco-search-services/{{ search.version }}/alfresco-search-services-{{ search.version }}.zip"
+  search_zip_sha1_checksum: "{{ lookup('url', '{{ nexus_repository.releases }}/alfresco-search-services/{{ search.version }}/alfresco-search-services-{{ search.version }}.zip.sha1', username=lookup('env', 'NEXUS_USERNAME'), password=lookup('env', 'NEXUS_PASSWORD')) }}"
+  sfs_jar_url: "{{ nexus_repository.enterprise_releases }}/alfresco-shared-file-store-controller/{{ sfs.version }}/alfresco-shared-file-store-controller-{{ sfs.version }}.jar"
+  sfs_jar_sha1_checksum: "{{ lookup('url', '{{ nexus_repository.enterprise_releases }}/alfresco-shared-file-store-controller/{{ sfs.version }}/alfresco-shared-file-store-controller-{{ sfs.version }}.jar.sha1', username=lookup('env', 'NEXUS_USERNAME'), password=lookup('env', 'NEXUS_PASSWORD')) }}"
+  trouter_jar_url: "{{ nexus_repository.enterprise_releases }}/alfresco-transform-router/{{ trouter.version }}/alfresco-transform-router-{{ trouter.version }}.jar"
+  trouter_jar_sha1_checksum: "{{ lookup('url', '{{ nexus_repository.enterprise_releases }}/alfresco-transform-router/{{ trouter.version }}/alfresco-transform-router-{{ trouter.version }}.jar.sha1', username=lookup('env', 'NEXUS_USERNAME'), password=lookup('env', 'NEXUS_PASSWORD')) }}"
+  transform_jar_url: "{{ nexus_repository.releases }}/alfresco-transform-core-aio-boot/{{ transform.version }}/alfresco-transform-core-aio-boot-{{ transform.version }}.jar"
+  transform_jar_sha1_checksum: "{{ lookup('url', '{{ nexus_repository.releases }}/alfresco-transform-core-aio-boot/{{ transform.version }}/alfresco-transform-core-aio-boot-{{ transform.version }}.jar.sha1', username=lookup('env', 'NEXUS_USERNAME'), password=lookup('env', 'NEXUS_PASSWORD')) }}"
```

Comment:
Not sure if we missed this one before or it regressed but this one shouldn't need nexus credentials as it's public